storage/api/datamanager/datamanager.go

```diff
@@ -0,0 +1,17 @@
+package datamanager
+
```

Comment:
For how to organize those, I perfer below:

```
|__ storage
|______ storage.go(interface)
|______ protos(group all proto files here)
|______ spanner
|__________ Implement spanner here
|______ TiKV...
```

Reply:
Roger that, will do.